src/live-preview.ts

```diff
@@ -339,9 +340,22 @@ export default class LivePreview {
 
                 if (this.config.ssr) {
                     // Get the content from the server and replace the body
-
-                    const fetch_url = new URL(window.location.href);
-
+                    const previewURL = sanitizeUrl(window.location.href);
+                    if (previewURL === "about:blank") {
+                        throw new Error(
+                            "LIVE PREVIEW SDK: Invalid URL " +
+                                window.location.href
+                        );
+                    }
```

Comment:
Are we catching this error somewhere? Let's ensure the entire app stays intact when someone adds a malicious URL.

Reply:
Removed the unused SSR code which include this section of code